src/components/header/style.ts

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+
+
+export const Container = styled.header`
+display: flex;
+justify-content: space-around;
+`;
+
+export const Image = styled.img`
```

Comment:
Are we don't need styles for image ? If no, lets remove this lines and use just img in header.tsx file